modules/shared/web/client/src/main/scala/web/client/table/package.scala

```diff
@@ -74,13 +88,16 @@ package table {
       val result = columns.toList.zipWithIndex.map {
         case (c @ ColumnMeta(_, _, _, true, PercentageColumnWidth(x)), idx) if idx === indexOf     =>
           val nv = x + delta
-          (nv >= 0 && nv <= 1).fold(c.copy(width = PercentageColumnWidth(nv)), c)
+          (nv >= 0 && nv <= 1)
+            .fold(c.copy(width = PercentageColumnWidth(nv)), c)
```

Comment:
Not a fan of boolean fold because it still typechecks if you write it backwards. I think we have argued about this before ;-)

Reply:
yeah, maybe i'm carried out by scala.js style that forces you to use it. I mean if you want to submit a PR to scala.js

Reply:
Oh scala.js uses it? Huh.

Reply:
I think I'll agree on this, especially as `Option.fold` works the other way around